parsers.js

```diff
@@ -0,0 +1,143 @@
+const _ = require("lodash");
+
+function object(value) {
+  if (_.isObject(value)) { return value; }
+  if (_.isString(value)) {
+    try {
+      return JSON.parse(value);
+    } catch (e) {
+      throw new Error(`Couldn't parse provided value as object: ${value}`);
+    }
+  }
+  throw new Error(`${value} is not a valid object`);
+}
+
+function number(value) {
+  const validNumber = (value) => _.isNumber(value) && _.isFinite(value) && !_.isNaN(value);
+  if (validNumber(value)) {
+    return value;
+  }
+  if (validNumber(parseFloat(value))) {
```

Comment:
What's wrong with saving the float in a const and reusing it?

Reply:
I am not sure, if I get what you mean.